tracing-test-utils/src/main/java/com/palantir/tracing/RenderTracingRule.java

```diff
@@ -1,64 +0,0 @@
-/*
- * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.tracing;
-
-import com.palantir.tracing.api.Span;
-import java.nio.file.Path;
-import java.util.Collection;
-import org.junit.rules.TestRule;
-import org.junit.runner.Description;
-import org.junit.runners.model.Statement;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public final class RenderTracingRule implements TestRule {
```

Comment:
This is no longer used by any of our code, and it's only references on public Github appear to be in outdated forks of some of our OSS repos.